davidbrucetn/nss-capstone-frontend-dwb
# Covidining user reg

# Description
Added User Settings to NavBar for affordance for user to change password. 

Fixes # (issue)
## Type of change
Please delete options that are not relevant.
User can now change password. It will update the json table and the session or local storage with the new value.

# Testing Instructions
Create a user and/or login and click on User Settings. Update your password to a new password. Logout and back in to verify.

# Checklist:
